File: Java_Codes/Decimal_to_Binary.java
import java.util.*;

public class Decimal_to_Binary {
    public static int decToBin(int n){
        StringBuilder sb = new StringBuilder("");

        while(n>=1){
            int a = n%2;
            sb.append(a);
            n /= 2;
        }
        for(int i=0; i<sb.length()/2; i++){
            char cr = sb.charAt(i);
            sb.setCharAt(i,sb.charAt(sb.length()-1-i));
            sb.setCharAt(sb.length()-1-i,cr);
        }
        return Integer.parseInt(sb.toString());
    }
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number and I'll give you it's decimal value");
        int n = sc.nextInt();
        System.out.println("The decimal value of " + n + " is : " + decToBin(n));
    }
}

File: python_codes/Decimal_to_binary.py
n = int(input("Enter a number :  "))
s = int(0)
count = int(0)
while n>0:
    a = n%2
    s = s*10+a
    if a==0 and s==0:
        count = count+1
    n = int(n/2)

b = 0
while s>0:
    a = int(s%10)
    b = b*10+a
    s = int(s / 10)
for i in range(0,count):
    b = b*10
    
print(b)